canonical/ubuntu.com
# Community member no of machines

## Done

- Change number of machines for community members to 50.
- @abhigyanghosh30 I moved the launchpad call to check for teams as part of the login. I noticed that the subscriptions took considerably longer to load if we do it when we call `/pro/user-subscriptions`. Even this with method login time is much longer. @jpmartinspt there should be ways to use `open_id` to query for team membership. But I did not want to spend much more time on this, did what we do for security api. 

Update: switched to check team membership as a OpenID exptension with the help of the logic from the django-openid-auth package we used for XYZ. Login is quick again.

## QA

- Fetch branch
- `dotrun`
- login you should see 5 (or 3) machines if you are not a community member
- change team to anything you are part of that's a public launchpad team fx: `xyz-admins`
- logout
- login
- you should see 50 machines now

## Issue / Card

Fixes #12453
